concordium-node/build.rs

```diff
@@ -119,13 +116,6 @@ fn main() -> std::io::Result<()> {
     #[cfg(feature = "static")]
     link_static_libs()?;
 
-    // build GRPC V1 interface.
-    tonic_build::configure()
-        .build_server(true)
-        .build_client(true) // the client is needed for the collector
-        .compile(&[&proto], &[&proto_root_input])
-        .expect("Failed to compile gRPC definitions!");
-
     build_grpc2(&proto_root_input)?;
```

Comment:
```suggestion
    build_grpc(&proto_root_input)?;
```
?

Reply:
No. I think that'll just cause more confusion.